modules/op-installing-pipelines-operator-in-web-console.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// */openshift_pipelines/installing-pipelines.adoc
+[id="op-installing-pipelines-operator-in-web-console_{context}"]
+= Installing the {pipelines-title} Operator in Web Console
+
+You can install {pipelines-title} using the Operator listed in the {product-name} OperatorHub. When you install the {pipelines-title} Operator, the custom resources required for the Pipelines configuration are automatically installed along with the Operator.
+
+[discrete]
+== Procedure
+
+. In the *Administrator* perspective of the Web Console, navigate to *Operators->OperatorHub*.
+
+. Use the *Filter by keyword* box to search for {pipelines-title} Operator in the catalog. Click on the *{pipelines-title} Operator* tile.
++
+image::op-installed-tile.png[]
+
+. Select the Operator, and click *Continue*.
```

Comment:
There is no `Continue`, clicking on tile shows the description

Reply:
done